Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -0,0 +1,2742 @@
+                'Weight': 'str',
+                'Location': 'str',
+                'CVSS Confidentiality Requirement': 'str',
+                'Technical Contact': 'str',
+                'Technical Owner': 'str',
+                'CVSS Collateral Damage Potential': 'str',
+                'Description': 'str',
+                'Business Owner': 'str',
+                'CVSS Integrity Requirement': 'str'
+            },
+            'id': 'int',
+            'domain_id': 'int',
+            'domain_name': 'str'
+        }
+    }
+    custom_fields = {
+        'events': {field.get('name'): field.get('property_type')
+                   for field in client.custom_properties()
+                   if 'name' in field and 'property_type' in field}
+    }
+    fields = {
+        'Offense': offense,
+        'Events: Builtin Fields': events,
+        'Events: Custom Fields': custom_fields,
+        'Assets': assets,
+    }
+    return fields
+
+
+def get_remote_data_command(client: Client, params: Dict[str, Any], args: Dict) -> GetRemoteDataResponse:
+    """
+    get-remote-data command: Returns an updated incident and entries
+
+    Args:
+        client (Client): QRadar client to perform the API calls.
+        params (Dict): Demisto params.
+        args (Dict):
+            id: Offense id to retrieve.
+            lastUpdate: When was the last time we data was retrieved in Epoch.
+
+    Returns:
+        GetRemoteDataResponse.
+    """
+    remote_args = GetRemoteDataArgs(args)
+    ip_enrich, asset_enrich = get_offense_enrichment(params.get('enrichment', 'IPs And Assets'))
+
+    offense = client.offenses_list(offense_id=remote_args.remote_incident_id)
+    offense_last_update = get_time_parameter(offense.get('last_persisted_time'))
+
+    # versions below 6.1 compatibility
+    last_update = get_time_parameter(args.get('lastUpdate'))
+    if last_update and last_update > offense_last_update:
+        demisto.debug('Nothing new in the ticket')
+        return GetRemoteDataResponse({'id': remote_args.remote_incident_id, 'in_mirror_error': ''}, [])
+
+    demisto.debug(f'Updating offense. Offense last update was {offense_last_update}')
+    entries = []
+    if offense.get('status') == 'CLOSED' and argToBoolean(params.get('close_incident', False)):
+        demisto.debug(f'Offense is closed: {offense}')
+        if closing_reason := offense.get('closing_reason_id', ''):
+            closing_reason = client.closing_reasons_list(closing_reason).get('text')
+
+        entries.append({
+            'Type': EntryType.NOTE,
+            'Contents': {
+                'dbotIncidentClose': True,
+                'closeReason': f'From QRadar: {closing_reason}'
+            },
+            'ContentsFormat': EntryFormat.JSON
+        })
+
+    enriched_offense = enrich_offenses_result(client, offense, ip_enrich, asset_enrich)
+    final_offense_data = sanitize_outputs(enriched_offense)[0]
+
+    demisto.debug(f'Pull result is {final_offense_data}')
```

Comment:
this could be huge no? why are we logging this?

Reply:
@Itay4 
it is only one offense, so shouldn't be too huge, but still you are right it isn't very small.
reason i added this is because I saw this is the behaviour in other implementations.
Can remove if you want